deploy/sre-prometheus/100-telemeter-client.PrometheusRule.yaml

```diff
@@ -11,8 +11,10 @@ spec:
   - name: sre-telemeter-client
     rules:
     - alert: MetricsClientSendFailingSRE
-      expr: absent(rate(metricsclient_request_send{status_code="200"}[5m]) > 0) or rate(metricsclient_request_send{status_code!="200"}[5m]) > 0
-      for: 15m
+      expr: 100 * rate(metricsclient_request_send{status_code="5xx"}[5m]) / rate(metricsclient_request_send[5m]) > 14.4 * 0.1 and 100 * rate(metricsclient_request_send{status_code="5xx"}[1h]) / rate(metricsclient_request_send[1h]) > 14.4 * 0.1
+      for: 5m
```

Comment:
@RiRa12621 Can you leave a comment explaining the formula and the numbers, especially where 14.4 is coming from?

Reply:
The 14.40 is from burn rate 2 within 1h and then up to the 30 day availability window. Thus `(2 * 24 * 28) / 100`.
Read more here: https://landing.google.com/sre/workbook/chapters/alerting-on-slos/

I also recently recorded a meeting in the SLO workgroup meeting where I explain most of this chapter. Let me know on Slack if you need the link.

Reply:
@metalmatze thanks for the clarification

Reply:
Please put that in the YAML file, too.

Reply:
or we can do `(2 * 24 * 30) / 100` in the yaml